### DISCUSSION TITLE
I am working on showing alert message using sweetalert2 in my  remix shopify app  but it didn't work

### DISCUSSION THREAD

**Discussion by pranto-rayple** [remix-run/remix#9499] [Q&A]
i install the package using 'npm install sweetalert2
'
here is my code.

`Swal.fire(
                        'Profile updated Successfully',
                        '',
                        'success'
                    );`

the message is showing as simple text bottom of the page

**Comment by Erij-Maherzia-BEN-BRAHIM** [comment]
Probably you forgot to import the css in the root @pranto-rayple
